client-data/tools/pencil/pencil.js

```diff
@@ -154,10 +154,12 @@
 			"icon": "tools/pencil/whiteout_tape.svg",
 			"active": false,
 			"switch": stopLine,
+			"size": -1
 		},
 		"mouseCursor": "url('tools/pencil/cursor.svg'), crosshair",
 		"icon": "tools/pencil/icon.svg",
-		"stylesheet": "tools/pencil/pencil.css"
+		"stylesheet": "tools/pencil/pencil.css",
+		"size": -1
```

Comment:
Having to specify `size: -1` in two distinct places in a tool to give it this ability is a little confusing. You should probably use the existing `secondary.switch` function to implement that.

Reply:
Fair point. One of the ideas behind two separate `size` attributes was one can specify different default sizes. I will look into more elegant implementation.